pkg/promclient/promclient.go

```diff
@@ -97,14 +97,17 @@ func NewWithTracingClient(logger log.Logger, userAgent string) *Client {
 	)
 }
 
-func (c *Client) get2xx(ctx context.Context, u *url.URL) (_ []byte, _ int, err error) {
+func (c *Client) get2xx(ctx context.Context, u *url.URL, additionalHeaders [][2]string) (_ []byte, _ int, err error) {
```

Comment:
```suggestion
func (c *Client) get2xx(ctx context.Context, u *url.URL, headers ...[2]string) (_ []byte, _ int, err error) {
```

That's kind of a point (:  to make those optional

Reply:
Oh, good point. Thanks! Let me update it.